app/assets/stylesheets/application.css.scss

```diff
@@ -59,6 +59,11 @@ pre.motd-monospaced {
   white-space: pre-wrap;
 }
 
+i.fa {
```

Comment:
Maybe this style should be labeled under a unique class, like `fa-center-14px`? As this will affect all Font Awesome icons in the Dashboard.